### ISSUE TITLE
Observation-wise losses for classification tasks

### ISSUE THREAD

**Issue by jemus42** [jemus42/xplainfi#51]
Generally, both Brier score and logloss should have observation-wise losses available but implementations in mlr3measures don't include them yet.

Unfortunately they're a bit more involved than adding a single line for a mapping like `obs_loss = "se"`, see https://github.com/mlr-org/mlr3measures/issues/57

As of now I think the only classif measure with an obs loss available is CE / Acc, which is better than nothing but when we want to showcase how we can do CPI for example, it would be nice to have at least logloss availaible.
